### ISSUE TITLE
Have 'Youtube Videos' tab in website

### ISSUE THREAD

**Issue by j-harbin** [ArgoCanada/argoFloats#80]
One quick question I have for now is, we chatted about getting a tab on the right of the 'home' page of the website for the youtube videos. I was thinking, A) if we added the table (photo attached) of the youtube videos to the README file, that could be one solution, as well as B) now that the table for videos is in the vignette, when you click on 'Getting Started' at  https://argocanada.github.io/argoFloats/index.html , you'll see there is a 'Youtube Videos' tab on the right. 

Thoughts?

Jaimie 

![image](https://user-images.githubusercontent.com/58750538/80606158-56df4400-8a0a-11ea-95d3-fc9d06e7f660.png)

**Comment by dankelley** [comment]
I added a sentence to the README file that says that the vignette has a list of youtube videos.  I prefer this to adding the list directly to the README file, because it is time-consuming and error-prone to maintain two lists with identical information.

Also, the listing of videos as a section in the vignette means that it appears on the website. Therefore, I do not think the website should be altered in the way that we had discussed.

In other words, I think this should be marked "done".  However, I am not the original reporter, so according to the convention of the project, I should not be the person to close the issue :-)

**Comment by dankelley** [comment]
PS. good work, Team!
